app/controlplane/configs/config.devel.yaml

```diff
@@ -58,3 +58,9 @@ auth:
 #   enabled: true
 #   allowed_orgs:
 #     - deadbeef
+
+onboarding:
```

Comment:
I'd not add this feature by default in development, otherwise we will not be able to test the `no orgs` available scenario. What do you think?